cli/paas/install_client.go

```diff
@@ -151,7 +151,11 @@ func (c *InstallClient) showInstallConfiguration(opts *kubernetes.InstallationOp
 func (c *InstallClient) fillInMissingSystemDomain(domain *kubernetes.InstallationOption) error {
 	if domain.Value.(string) == "" {
 		ip := ""
-		if !c.kubeClient.GetPlatform().HasLoadBalancer() {
+		foundLoadBalancer, err := c.kubeClient.GetPlatform().HasLoadBalancer(c.kubeClient.Kubectl)
```

Comment:
What do you think about showing a message that a loadbalancer was found? As general information and also for debugging purposes?

Reply:
Removed this part of the code due to change in logic